File: Modul-1/Soal-4/PRAK104-2410817210008-AhmadUlyani.c
#include <stdio.h>

int main()
{
    int A = 400000;
    int B = 350000;
    float diskonA = 0.13*A;
    float hargasetelahdiskonA = A - diskonA;
    float diskonB = 0.21*B;
    float hargasetelahdiskonB = B - diskonB;

    printf("Harga sepatu A adalah %d\n", A);
    printf("Harga sepatu B adalah %d\n", B);
    printf("Sepatu A mendapatkan diskon 13%% sehingga harganya menjadi %.0f\n", hargasetelahdiskonA);
    printf("Sepatu B mendapatkan diskon 21%% sehingga harganya menjadi %.0f\n", hargasetelahdiskonB);

    return 0;
}

File: Modul-1/Soal-4/PRAK104-2410817210008-AhmadUlyani.py
A = 400000
B = 350000

print("Harga sepatu A adalah", A)
print("Harga sepatu B adalah", B)

diskonA = 0.13*A
hargasetelahdiskonA = A - diskonA
diskonB = 0.21*B
hargasetelahdiskonB = B - diskonB

print(f"Sepatu A mendapatkan diskon 13% sehingga harganya menjadi {hargasetelahdiskonA:.0f}")
print(f"Sepatu B mendapatkan diskon 21% sehingga harganya menjadi {hargasetelahdiskonB:.0f}")